Ecosystem Partners/Marketplace Guides/getting-started-with-pivotal-cloud-foundry.md

```diff
@@ -71,6 +72,9 @@ Pivotal Cloud Foundry is a complex piece of software.  Before installing please
 
   Starting from the CenturyLink Control Panel, navigate to the Blueprints Library. Search for "Pivotal Cloud Foundry" in the keyword search on the right side of the page.
 
+  > Note for access to this Blueprint during this early access period
+  > # Email noc@ctl.io
```

Comment:
Yes.

From: Nathan Young
Reply-To: CenturyLinkCloud/PublicKB
Date: Tuesday, August 4, 2015 at 3:52 PM
To: CenturyLinkCloud/PublicKB
Cc: Keith Resar
Subject: Re: [PublicKB] ecosystem / pcf img ref (#268)


In Ecosystem Partners/Marketplace Guides/getting-started-with-pivotal-cloud-foundry.md<https://github.com/CenturyLinkCloud/PublicKB/pull/268#discussion_r36240173>:

> @@ -71,6 +72,9 @@ Pivotal Cloud Foundry is a complex piece of software.  Before installing please
>
>    Starting from the CenturyLink Control Panel, navigate to the Blueprints Library. Search for "Pivotal Cloud Foundry" in the keyword search on the right side of the page.
>
> +  > Note for access to this Blueprint during this early access period
> +  > # Email noc@ctl.io<mailto:noc@ctl.io>


is this h1 inside a block quote intentional?

—
Reply to this email directly or view it on GitHub<https://github.com/CenturyLinkCloud/PublicKB/pull/268/files#r36240173>.